src/common/components/IconBox/IconBox.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, ChakraProps } from '@chakra-ui/react';
+import React, { PropsWithChildren } from 'react';
+import { RoundIcon } from './RoundIcon';
+
+interface IconBoxProps extends ChakraProps {
```

Comment:
interface IconBoxProps extends IconProps {

lets use IconProps

Reply:
Actually its using BoxProps, let me change that